coreos-base/hard-host-depends/hard-host-depends-0.0.1.ebuild

```diff
@@ -148,11 +130,6 @@ RDEPEND="${RDEPEND}
 	dev-python/gdata
 	"
 
-# Host dependencies for HWID processing
-RDEPEND="${RDEPEND}
-	dev-python/pyyaml
```

Comment:
Should this package be dropped?

Reply:
I _think_ so. Everything built fine without it. `hard-host-depends` is the only thing that depends on it and `repo grep "import yaml" ` and `repo grep "from yaml"` don't turn anything up. [git blame](https://github.com/coreos/coreos-overlay/blame/master/coreos-base/hard-host-depends/hard-host-depends-0.0.1.ebuild#L153) shows it's from the initial import from ChromeOS